stdlib.sh

```diff
@@ -533,19 +707,19 @@ function __STDLIB_API_1_std::wrap() { # {{{
 
 	if [[ -n "${prefix:-}" ]]; then
 		if (( columns > ( ${#prefix} + 2 ) )); then
-			  output "${text}" \
+			  output "${outputFlag}" "${text}" \
```

Comment:
To ensure that `outputFlag` (urg, camel-case) isn't used uninitialised, use `${outputFlag:-}`...